src/shared/Notifications/Notifications.js

```diff
@@ -1,67 +1,120 @@
 import React, { Component } from 'react';
 import cx from 'classnames';
 import PropTypes from 'prop-types';
-
-import { NOTIFICATION_TYPES } from '../../constants';
+// import { NOTIFICATION_TYPES } from '../../constants';
 
 import './Notifications.scss';
 
 export const CN = 'notifications';
 
+const notificationIcons = {
+  info: {
+    type: 'info',
+    icon: 'info icon'
+  },
+  success: {
+    type: 'success',
+    icon: 'check circle '
+  },
+  warning: {
+    type: 'warning',
+    icon: 'exclamation triangle'
+  },
+  error: {
+    type: 'error',
+    icon: 'x icon'
+  },
+  change: {
+    type: 'change',
+    icon: 'pencil alternate'
+  },
+  custom: {
+    type: 'custom',
+    icon: 'wrench'
+  }
+};
+
 class Notifications extends Component {
   constructor(props) {
     super(props);
 
-    this.hideMessage = this.hideMessage.bind(this);
+    this.hideNotification = this.hideNotification.bind(this);
+    this.renderNotifications = this.renderNotifications.bind(this);
+    this.renderIcon = this.renderIcon.bind(this);
   }
 
-  hideMessage() {
-    const { isHideNotification } = this.props;
-    isHideNotification();
+  hideNotification(id) {
+    const { removeItemNotification } = this.props;
+    removeItemNotification(id);
   }
 
-  render() {
-    const {
+  renderIcon(type) {
+    if (type === notificationIcons.info.type) {
```

Comment:
In this case better to rewrite to switch-case

Reply:
done)